static/src/js/selectors/preferences.js

```diff
@@ -46,3 +47,24 @@ export const getGranuleSortPreference = createSelector(
     return granuleSort
   }
 )
+
+/**
+ * Retrieve the parameter collection sort key from Redux
+ * If it is the same as the user preference sort key, return null.
+ * This function is used for inserting the sortKey
+ * into the URL query parameters when a user selects
+ * a different collection sorting key.
+ * @param {Object} state Current state of Redux
+ */
+export const getCollectionSortKeyParameter = (state) => {
+  const { query } = state
+  const { collection } = query
+  const { paramCollectionSortKey } = collection
+  const userPrefSortKey = translateDefaultCollectionSortKey(getCollectionSortPreference(state))
+
+  if (paramCollectionSortKey === userPrefSortKey) {
```

Comment:
```suggestion
// Do not show url parameter if preference matches the current query
  if (paramCollectionSortKey === userPrefSortKey) {
```

Reply:
We'll update this later when we change it so that it would show up for all user preferences